carveracontroller/update_translations.py

```diff
@@ -0,0 +1,55 @@
+####
+#this program builds the translation files automatically from the main python programs
+#it requires gettext https://www.gnu.org/software/gettext/
+#on windows this requires installing https://gnuwin32.sourceforge.net/packages/gettext.htm
+
+#to add a new language, update the LANGS array in main.py with the two letter code and the display name. 
+
+#run with python update_translations.py while in the carveracontroller directory
+
+
+####
+
+
+import subprocess
+import os
+
+# Generate the .pot file from source files. Add any new UI source files here
+# Separate .kv files to use the --language=Python option
+python_files = ["main.py", "controller.py", "gcodeViewer.py"]  # Python source files
+kv_files = ["makera.kv"]  # .kv files
+
+pot_file = "locales/messages.pot"
+subprocess.run(["xgettext", "-d", "messages", "-o", pot_file, "--from-code=UTF-8"] + python_files)
+print(f"Generated .pot file from Python files: {pot_file}")
+
+# Process .kv files separately with --language=Python
+for kv_file in kv_files:
+    subprocess.run(["xgettext", "-j", "-d", "messages", "-o", pot_file, "--from-code=UTF-8", "--language=Python", kv_file])
+    print(f"Appended .pot file with entries from {kv_file}")
+
+# List of languages for .po files
+languages = ["de", "en", "es", "fr", "it", "ja", "ko", "pt", "zh-CN", "zh-TW"]
```

Comment:
Let's remove all but the currently supported languages. Both from the script and also from the carveracontroller/langs/ folder. 

```suggestion
languages = ["en","zh-CN"]
```